pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["setuptools", "setuptools_scm", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fmu-steaclient"
+authors = [
+  {name = "Equinor ASA", email = "fg_sib-scout@equinor.com"},
+]
+description="stea - calculate economic analysis"
+requires-python = ">=3.8"
+readme = "README.md"
+classifiers=[
+        "Development Status :: 1 - Planning",
+        "Environment :: Plugins",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+        "Natural Language :: English",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dynamic = ["version"]
+dependencies=["requests", "pyyaml", "pydantic>=2", "resdata", "click"]
+
+[project.urls]
+repository = "https://github.com/equinor/fmu-steaclient"
+
+[project.entry-points."ert"]
+stea_job = "stea.fm_stea.fm_stea"
+
+[project.entry-points."console_scripts"]
+fmu_steaclient = "stea.fm_stea.fm_stea:main_entry_point"
+
+[tool.setuptools_scm]
+write_to = "stea/version.py"
+
+[project.optional-dependencies]
+ert = ["ert"]
+test = [
+"fmu-steaclient[ert]",
+"pytest",
+"pytest-mock",
+"pytest-httpserver",
+]
+
+[tool.black]
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.tox
+    | \.venv
+    | \stea/version.py
```

Comment:
For some reasons this didn't get excluded. chatgpt suggested: `stea/version\.py`